.gitignore

```diff
@@ -1,3 +1,7 @@
 node_modules
 dist
-yarn-error.log
\ No newline at end of file
+
+index.html
```

Comment:
Why do you exclude `index.html` from the git repository?

Reply:
That's wrong. Don't remember why I did that..

Reply:
You should do self-review before you create the pull request on Github. I think you can catch this obvious problem easily. :D